### ISSUE TITLE
Upload control remove file button is not keyboard accessible

### ISSUE THREAD

**Issue by yuvrajsawant-github** [ant-design/ant-design#24519]
![upload_removebutton](https://user-images.githubusercontent.com/64444288/83078489-98c1d100-a02e-11ea-9860-02fade430efa.PNG)
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/sfbuw?file=/index.js)

### Steps to reproduce
1. Upload any file in the upload control 
2. Use keyboard "tab" key to navigate to remove file button for the uploaded file
3. It seems that unable to remove file using keyboard "tab" key and the focus not set on uploaded file

### What is expected?
Upload control should be keyboard accessible 

### What is actually happening?
Unable to focus the remove file button on uploaded file

| Environment | Info |
|---|---|
| antd | 4.2.5 |
| React | 16.12.0 |
| System | Windows 10 |
| Browser | Edge Version 81.0.416.72 (Official build) (64-bit) or Chorme Version 81.0.4044.138 (Official Build) (64-bit) |
